examples/vision/supervised-contrastive-learning.py

```diff
@@ -0,0 +1,204 @@
+"""
+Title: Supervised Contrastive Learning
+Author: [Khalid Salama](https://www.linkedin.com/in/khalid-salama-24403144/)
+Date created: 2020/11/01
+Last modified: 2020/11/01
+Description: Using supervised contrastive learning for image classification.
+"""
+
+"""
+## Introduction
+
+[Supervised Contrastive Learning](https://arxiv.org/abs/2004.11362)
+(Prannay Khosla et al.) is a training methodology that outperforms cross-entropy
+on supervised learning tasks.
+
+Essentially, training an image classification model with Supervised Contrastive Learning
+is peformed in two phases:
+
+  1. Pre-training an encoder to generate feature vectors for input images such that feature
```

Comment:
Unindent the list so that it renders properly

Reply:
Done